ui/src/components/layout/FlowCanvas.tsx

```diff
@@ -16,21 +17,21 @@ import AgentNode from '../nodes/AgentNode';
 import ResourceSpecificationNode from '../nodes/ResourceSpecificationNode';
 import getDataStore from "../../data/store";
 import ModalContainer from '../modals/ModalContainer';
-import { DisplayNode, DisplayEdge, PathedData, ObjectTransformations, ObjectTypeMap } from "../../types/valueflows";
+import { DisplayNode, ObjectTransformations, ObjectTypeMap } from "../../types/valueflows";
```

Comment:
Technically, DisplayEdge could stick around, but it's easy enough for our IDEs to automatically bring it back when we uncomment the code that uses it below.